content-override/en/_index.html

```diff
@@ -0,0 +1,120 @@
+						implementations, Knative solves the "boring but difficult" parts of 
+						building, deploying, and managing cloud native services so you don't
+						have to.
+					</p>
+					<div>&nbsp;</div>
+					<div>&nbsp;</div>
+					<h3>Highlights</h3>
+					<ul>
+						<li>Focused API with higher level abstractions for common app use-cases.</li>
+						<li>Stand up a scalable, secure, stateless service in seconds.</li>
+						<li>Loosely coupled features let you use the pieces you need.</li>
+						<li>Pluggable components let you bring your own logging and monitoring, networking, and service mesh.</li>
+						<li>Knative is portable: run it anywhere Kubernetes runs, never worry about vendor lock-in.</li>
+						<li>Idiomatic developer experience, supporting common patterns such as
+							GitOps, DockerOps, ManualOps. Knative can be used with common tools and
+							frameworks such as Django, Ruby on Rails, Spring, and may more.
+						</li>
+					</ul>
+					<div>&nbsp;</div>
+					<div>&nbsp;</div>
+				</div>
+			</div>
+	</div>
+</section>
+
+<section class="row td-box td-box--1" style="margin-top: -80px">
+		<div class="col">
+			<div class="row">
+				<div class="col-12 ">
+					<h3 class="d-flex justify-content-center">Knative Features</h3>
+					<br>
+				</div>
+			</div>
+			<div class="row justify-content-center">
+				<div class="col-sm-12 col-md-4 col-lg-3">
+					<div class="d-flex justify-content-center"><h4>Build</h4></div>
+					<p>Provides easy-to-use, simple source-to-container builds, so you can focus on writing code and now how to build it. Knative solves for the common challenges of building containers and runs it on cluster.</p>
+				</div>
+				<div class="col-sm-12 col-md-4 col-lg-3">
+						<div class="d-flex justify-content-center"><h4>Serving</h4></div>
+						<p>Run serverless containers on Kubernetes with ease, Knative takes care of the details of networking, autoscaling (even to zero), and revision tracking. You just have to focus on your core logic.</p>
+				</div>
+				<div class="col-sm-12 col-md-4 col-lg-3">
+						<div class="d-flex justify-content-center"><h4>Eventing</h4></div>
+						<p>Universal subscription, delivery, and management of events. Build
+							modern apps by attaching compute to a data stream with declarative event
+						connectivity and developer-friendly object model. </p>
+				</div>
+			</div>
+	</div>
+</section>
+
+<section class="row td-box td-box--2">
+		<div class="col">
+			<div class="row justify-content-center">
+					<div class="col-sm-12 col-md-4 col-lg-3 text-center">
+							<div class="mb-4 h1">
+									<i class="fab fa-slack"></i>
+								</div>
+								<h4 class="h3">Talk to us on Slack</h4>
+								<p class="mb-0"></p><p>Interested in learning more, speaking to other contributors, or finding answers?</p>
+							<p></p>
+							<p><a href="https://knative.slack.com/">Read More …</a></p>
+					</div>
+					<div class="col-sm-12 col-md-4 col-lg-3 text-center">
```

Comment:
Just noticed that our licensing does not show on the main page, do we want (or do we need to) list those here too (today they only show in footers on the individual content pages)?

License params are here: https://github.com/knative/website/blob/master/config/_default/params.toml#L4

You would be able to call those individual values by prefixing a parameter name with `.Site.Params.` for example:

`{{ .Site.Params.commons_License }}`
`{{ .Site.Params.commonsLicense }}`

`<a href="{{ .Site.Params.commons_License }}">{{ .Site.Params.commonsLicense }}</a>`

This is another thing that I could help with if its needed (ie. we could merge this and open another PR to add the common and apache licenses)

Reply:
Good call. We should add this for sure. I probably won't have time to work on it this week, you could either edit this PR directly, or we could approve this PR and merge it, and then make these edits immediately afterward.

Reply:
Ok, yeah, lets "ship and iterate". Ill open another PR to add licensing to the footer.